hops/job.py

```diff
@@ -0,0 +1,61 @@
+import json
+import sys
+
+from hops import util
+from hops import constants
+from hops.exceptions import RestAPIError
+
+BASE_API = "/hopsworks-api/api"
+RUN_JOB = ("POST", BASE_API +
+           "/project/{project_id}/jobs/{job_name}/executions?action=start")
+# Get the latest execution
+EXECUTION_STATE = ("GET", BASE_API +
+                   "/project/{project_id}/jobs/{job_name}/executions?sort_by=appId:desc&limit=1")
+
+
+def launch_job(job_name):
+    """
+    Function for launching a job to Hopsworks. 
+
+    :param job_name: Name of the job to be launched in Hopsworks
```

Comment:
Can you change the docstrings in `job.py` to be consistent with the others? If you use pycharm you can configure it to use 'Google' docstring format by default, which is the one we are using. (add Returns-info also, since it will be rendered by Sphinx later)
![docstring](https://user-images.githubusercontent.com/8254791/65749748-08319a80-e107-11e9-8e14-0239aa889b57.png)

Reply:
done